zproject/test_extra_settings.py

```diff
@@ -175,6 +175,7 @@ def set_loglevel(logger_name: str, level: str) -> None:
 SOCIAL_AUTH_APPLE_SERVICES_ID = 'com.zulip.chat'
 SOCIAL_AUTH_APPLE_BUNDLE_ID = 'com.zulip.bundle.id'
 SOCIAL_AUTH_APPLE_CLIENT = 'com.zulip.chat'
+SOCIAL_AUTH_APPLE_AUDIENCE: List[Optional[str]] = [SOCIAL_AUTH_APPLE_BUNDLE_ID, SOCIAL_AUTH_APPLE_SERVICES_ID]
```

Comment:
This is not correct yet, having a `None` in this list is not something we want. The type should be `List[str]` and we should make mypy happy with it (by correctly defining this list in computed_settings - in a way the ensures mypy there are no `None` in there).